modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,6 +1,8 @@
 @import "mixins";
 
 .portlet-asset-publisher {
+	word-wrap: break-word;
+
 	.asset-full-content {
```

Comment:
Move this under `.asset-full-content`

Reply:
I originally tried putting it under there but it didn't change the wrapping style, so I placed it in its parent.

Reply:
I believe `portlet-asset-publisher` is going to be too general, and apply to additional elements we are not intending. Can you see if there's another common ancestor under `.portlet-asset-publisher` that would be a good fit? Maybe something like `.asset-content`.

Reply:
I understand, I'll look around!

Reply:
I couldn't find any common ancestor under `.portlet-asset-publisher` that applies the fix. I also searched the project for `asset-content` and `asset-summary`, but there wasn't any relevant findings that would apply the fix. Would it be okay to create `.asset-content` as a child of `.portlet-asset-publisher` and apply the word-wrap property within there?

Reply:
Can you paste DOM between `<div class="portlet-asset-publisher">` and the first content you want to style?

```html
<div class="portlet-asset-publisher">
    <div class="some-ancestor with-some-class">
        <div class="asset-summary">
            <div class="asset-content">
                <p>My content HTML</p>
```

Reply:
The highlighted blue to the "ccc...".
![dom_asset](https://user-images.githubusercontent.com/46464263/57025528-32307080-6bec-11e9-9ab6-5054fcc171c2.png)

Reply:
Targeting `.asset-content` should work just fine in this case.

So you could use
```css
.asset-content, .asset-full-content {
```